### PR TITLE
Test: Add frozen funds for 6 more validators

### PR DESCRIPTION
**Pull request by omerfirmak**

### CODE REVIEW

File: source/agora/utils/WellKnownKeys.d [hunk 1]
```diff
@@ -1190,6 +1190,24 @@ static immutable NODE6 = KeyPair(PublicKey(Point([217, 223, 234, 237, 60, 4, 220
...
+
+// vala: boa1xqvala342upf2t2c0fe96h8cdtjatrksjldyhsrachcq3523ah8dy5r8ejz - SDDEBCVMX7GDWWEENJFAJCSLMDWXYCGMIGXPS6NDE2R7ZISICTTAYIMR
+static immutable NODE10 = KeyPair(PublicKey(Point([25, 223, 246, 53, 87, 2, 149, 45, 88, 122, 114, 93, 92, 248, 106, 229, 213, 142, 208, 151, 218, 75, 192, 125, 197, 240, 8, 209, 81, 237, 206, 210])), SecretKey(Scalar([198, 64, 138, 172, 191, 204, 59, 88, 132, 106, 74, 4, 138, 75, 96, 237, 124, 8, 204, 65, 174, 249, 121, 163, 38, 163, 252, 162, 72, 20, 230, 12])));
+
+// valc: boa1xqvalc7v34kr9crh4e882zmguvt3dgmtdhxtqx0wsljej5f9xdxl6xftcay - SAWUJXYVIHQ2TKT4WXZ7XP5PFMSIN5HZI6CZWAMOAEKZKCOLIHSQM2DM
+static immutable NODE11 = KeyPair(PublicKey(Point([25, 223, 227, 204, 141, 108, 50, 224, 119, 174, 78, 117, 11, 104, 227, 23, 22, 163, 107, 109, 204, 176, 25, 238, 135, 229, 153, 81, 37, 51, 77, 253])), SecretKey(Scalar([45, 68, 223, 21, 65, 225, 169, 170, 124, 181, 243, 251, 191, 175, 43, 36, 134, 244, 249, 71, 133, 155, 1, 142, 1, 21, 149, 9, 203, 65, 229, 6])));
+
+// vald: boa1xrvaldd5au5d5xs6pd6js7zah6m4h5d0r5tpwjasp99gvz3gmj2ex432u5x - SDSJTGPAHSOO3A4GO4USKQTMXPQFA6YBZFQWKFQX2L3FNQ47PJ3QQ77H
```
Comment (Geod24) [inline source/agora/utils/WellKnownKeys.d:1205]: Well what I tried to do is to have a number following a letter, or the other way around, so it's easier to parse from a quick look.

**Comment by Geod24** [comment]
I expect most tests will fail though.

File: source/agora/utils/WellKnownKeys.d (hunk 1 above)
Comment (hewison-chris) [reply source/agora/utils/WellKnownKeys.d:1205]: Not so easy when `1` is not a valid character.

**Review by hewison-chris — APPROVED** [review]
LGTM, I set off another ci run to see if it goes green.

**Pull request merged**